sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/src/instrumenter.ts

```diff
@@ -21,11 +28,26 @@ export class OpenTelemetryInstrumenter implements Instrumenter {
     name: string,
     spanOptions: InstrumenterSpanOptions
   ): { span: TracingSpan; tracingContext: TracingContext } {
-    const span = trace
-      .getTracer(spanOptions.packageName, spanOptions.packageVersion)
-      .startSpan(name, toSpanOptions(spanOptions));
-
     let ctx = spanOptions?.tracingContext || context.active();
+    let span: Span;
+
+    if (environmentVariableToBoolean("AZURE_TRACING_DISABLED")) {
```

Comment:
reading env var is usually not cheap and they also don't really change at runtime, so caching the value in a static variable can result in great perf improvements.

Reply:
Yeah that's a really good point, I can cache them 👍